Packs/AWS-GuardDuty/ReleaseNotes/1_2_9.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### AWS - GuardDuty
+- Documentation and metadata improvements.
```

Comment:
In general (and here too) we want to refrain from writing this sentence as it confuses customers. Instead, let's give a very short summary of the stuff we added.

Reply:
Btw, this is generated by default in the update-release-notes command in the SDK

Reply:
What do you mean by default?

Reply:
When running `demisto-sdk update-release-notes -i Packs/pack-name -u documentation` this is generated automatically to the RN